### PR TITLE
Add logging test that supports ES and AWS CloudWatch

### PR DESCRIPTION
**Pull request by dry923**
# Description
Adds support for running logging tests. This works for both Elasticsearch backed and AWS CloudWatch backed configurations.

### CODE REVIEW

File: workloads/logging/common.sh [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+  done
+  log "Benchmark in progress"
+  until oc get benchmark -n my-ripsaw log-generator-${UUID} -o jsonpath="{.status.state}" | grep -Eq "Complete|Failed" | wc -l -eq $POD_COUNT; do
+    if [[ $(date +%s) -gt ${timeout} ]]; then
+      log "Timeout waiting for Benchmark to complete"
+      exit 1
+    fi
+    sleep 1
+  done
+  status=$(oc get benchmark -n my-ripsaw log-generator-${UUID} -o jsonpath="{.status.state}")
```
Comment (whitleykeith) [inline workloads/logging/common.sh:175]: see line 148

**Review by whitleykeith — CHANGES_REQUESTED** [review]

**Review by whitleykeith — APPROVED** [review]
lgtm!

**Comment by dry923** [comment]
The ES portion has been tested. The CloudWatch backend has not yet. We can treat the aws portion as a WIP and merge this sooner if needed for the ES part.

**Comment by whitleykeith** [comment]
i'm gonna merge since the ES option is stable

**Pull request merged**